src/sirinternal.c

```diff
@@ -1534,11 +1551,11 @@ long __sir_nprocs(bool test_mode) {
         nprocs = vtprocs;
 #endif
 
-    if (nprocs < 1) {
-        _sir_selflog(BRED("Failed to determine processor count!"));
+    if (nprocs < 1) { // GCOVR_EXCL_START
```

Comment:
adding GCOVR_EXCL_START doesn't make us have more code coverage, it just makes it look like we do. it's stupid.